revolv/base/views.py

```diff
@@ -59,9 +60,25 @@ def get_context_data(self, **kwargs):
         context["project_dict"] = project_dict
         context["role"] = self.role or "donor"
 
+        context['category_setter_url'] = reverse('dashboard_category_setter')
+        context['categories'] = Category.objects.all().order_by('title')
```

Comment:
This is why the categories don't show up: this is the base view only for the administrator and ambassador dashboards (as the docstring says). You'll need to add the context data to the donor dashboard view too (I think it's in donor/views)